tests/transforms/test_qcut.py

```diff
@@ -5090,7 +5097,9 @@ def test_find_and_place_cuts(self, local_measurement, with_manual_cut, cut_strat
             qml.CNOT(wires=["a", "b"])
             qml.RX(0.5, wires="a")
             qml.RY(0.6, wires="b")
-            qml.expval(qml.PauliX(wires=[0]) @ qml.PauliY(wires=["a"]) @ qml.PauliZ(wires=["b"]))
+            qml.expval(
+                qml.prod(qml.PauliX(wires=[0]), qml.PauliY(wires=["a"]), qml.PauliZ(wires=["b"]))
```

Comment:
Why are we using `qml.prod` here instead of `@`? This seems like another test case where we might also want to test legacy behaviour as well.

Reply:
Yep I was doing that